CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# Changelog
+
+Changelog for K8ssandra, new PRs should update the `main / unreleased` section with entries in the order:
+
+```markdown
+* [CHANGE]
+* [FEATURE]
+* [ENHANCEMENT]
+* [BUGFIX]
+```
+
+When cutting a new release of the parent `k8ssandra` chart update the `main / unreleased` heading to the tag being generated and date `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `main / unreleased` entries.
+
+## main / unreleased
+
+* [FEATURE] #409 Add support for `configOverride`
```

Comment:
Are these issue numbers or PR numbers? When adding a new PR, should we create an entry at the top or the bottom of the unreleased section?

Here's an entry for my in-flight PR:
```suggestion
* [ENHANCEMENT] #436 Upgrade Stargate to 1.0.11 and add a `preStop` lifecycle hook to improve behavior when reducing the number of Stargate replicas in the presence of live traffic
* [FEATURE] #409 Add support for `configOverride`
```

Reply:
It is currently the PR number. The order outside of the category seems less important. We could try and do it by when it was merged in to main, or based on the PR number. What are your thoughts?

Reply:
For your inflight PR do you want to rebase off of main and update the CHANGELOG there (in the PR)?